docs/config.rst

```diff
@@ -1,11 +1,606 @@
+
+
+Add a Flow Step
+^^^^^^^^^^^^^^^
+
+If you want to add a step to a flow, use ``cci flow info <name>`` first to see the existing steps.
+
+    Example: ``dev_org``
+
+.. code-block:: console
+
+    $ cci flow info dev_org
+    Description: Set up an org as a development environment for unmanaged metadata
+    1) flow: dependencies [from current folder]
+        1) task: update_dependencies
+        2) task: deploy_pre
+    2) flow: deploy_unmanaged
+        0) task: dx_convert_from
+        when: project_config.project__source_format == "sfdx" and not org_config.scratch
+        1) task: unschedule_apex
+        2) task: update_package_xml
+        when: project_config.project__source_format != "sfdx" or not org_config.scratch
+        3) task: deploy
+        when: project_config.project__source_format != "sfdx" or not org_config.scratch
+        3.1) task: dx_push
+            when: project_config.project__source_format == "sfdx" and org_config.scratch
+        4) task: uninstall_packaged_incremental
+        when: project_config.project__source_format != "sfdx" or not org_config.scratch
+    3) flow: config_dev
+        1) task: deploy_post
+        2) task: update_admin_profile
+    4) task: snapshot_changes
+
+There are four steps to this flow. The first three steps are themselves flows, and the last step is a task.
+
+All *non-negative numbers and decimals* are valid as step numbers in a flow.
+You can add steps before, in-between, or after existing flow steps.
+If you want to add a step at the beginning of the dev org flow, valid step numbers would include any number greater than or equal to zero and less than 1. Example values would include 0, 0.3, and 0.89334. All of these would cause the step to execute before step 1 in the ``dev_org`` flow.
+
+If you want to add a step *between* steps 2 and 3, then a step number greater than 2 or less than 3 can be used.
+
+If you want to add a step *after* all steps in the flow, then any step number greater than 4 can be used.
+
+You can also add an additional log line output during the execution of a flow.
+
+    Example: ``dev_org``
+
+.. code-block:: yaml
+
+    dev_org:
+        steps:
+            5:
+                task: log
+                    options:
+                        line: dev_org flow has completed
+
+
+Skip a Flow Step
+^^^^^^^^^^^^^^^^
+
+To skip a flow step, set the desired step number to a task or flow with the value of ``None``.
+
+    Example: To skip the fourth step of the ``dev_org`` flow, insert the following under the ``flows`` section of your ``cumulusci.yml`` file.
+
+.. code-block:: yaml
+
+    dev_org:
+        steps:
+            4:
+                task: None
+
+.. note::
+    The key of ``task`` must be used when skipping a flow step that is a task. The key of ``flow`` must be used when skipping a flow step that corresponds to a flow.
```

Comment:
```suggestion
    If skipping a flow step that is a task, always use the ``task`` key. If skipping a flow step that is a flow, always use the `flow`` key.
```
Rewording to avoid passive and wordiness. Here's a new style entry for you: https://confluence.internal.salesforce.com/display/DOCTEAM/need+to%2C+have+to%2C+must